de.metas.swat/de.metas.swat.base/src/main/java/de/metas/request/service/IRequestCreator.java

```diff
@@ -0,0 +1,41 @@
+package de.metas.request.service;
+
+import java.util.List;
+import java.util.Properties;
+
+import org.adempiere.util.ISingletonService;
+
+/*
+ * #%L
+ * de.metas.swat.base
+ * %%
+ * Copyright (C) 2016 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+public interface IRequestCreator extends ISingletonService
+{
+
+	/**
+	 * Create an R_Request based on a list of M_InOutLine IDs
+	 * 
+	 * @param ctx
+	 * @param inOutLineIds
+	 * @param trxName
+	 */
+	void createRequest(final Properties ctx, final List<Integer> docLineIds, final String trxName);
```

Comment:
btw, what happens if someone calls this with a bunch of C_OrderLine_IDs? are there any checks down the road? did you consider adding I_M_InOutLines instead of the IDs?